src/ontology/uberon.Makefile

```diff
@@ -204,6 +204,7 @@ $(EDIT_PREPROCESSED): $(SRC)
 		reason -r elk --exclude-duplicate-axioms true --equivalent-classes-allowed asserted-only \
 		unmerge -i $(TMP_REFL) \
 		unmerge -i $(DEVELOPS_FROM_CHAIN) \
+    remove --axioms type \
```

Comment:
Something seems missing here.. What does `type` do?

Reply:
Remove the declaration you said to remove.

Reply:
Sweet, thanks. Yes, thats great.